### PR TITLE
Add guides by Somer Esat

### PR DESCRIPTION
**Pull request by sandakersmann**
Add guides by Somer Esat

### CODE REVIEW

File: src/content/developers/docs/nodes-and-clients/run-a-node/index.md [hunk 1]
```diff
@@ -460,6 +460,7 @@ As part of your monitoring, make sure to keep an eye on your machine's performan
 ## Further reading {#further-reading}
 
 - [Guide | How to setup a validator for Ethereum staking on mainnet](https://www.coincashew.com/coins/overview-eth/guide-or-how-to-setup-a-validator-on-eth2-mainnet) _– CoinCashew, updated regularly_
+- [Ethereum Staking Guides](https://github.com/SomerEsat/ethereum-staking-guides) _– Somer Esat, updated regularly_
 - [ETHStaker guides on running validators on testnets](https://github.com/remyroy/ethstaker#guides) – _ETHStaker, updated regularly_
```
Comment (minimalsm) [inline src/content/developers/docs/nodes-and-clients/run-a-node/index.md:464]: [As per our framework for product ordering on our listing criteria](https://ethereum.org/en/contributing/adding-products/#product-ordering), this should go at the bottom of the 'staking guides'.

```suggestion
- [ETHStaker guides on running validators on testnets](https://github.com/remyroy/ethstaker#guides) – _ETHStaker, updated regularly_
- [Ethereum Staking Guides](https://github.com/SomerEsat/ethereum-staking-guides) _– Somer Esat, updated regularly_
```

Comment (minimalsm) [reply src/content/developers/docs/nodes-and-clients/run-a-node/index.md:464]: Meta comment: this page is about running a node, so the node running further reading should probably be placed above the staking further reading (should we even have staking information here vs pointing people to /staking/?)

cc: @jmcook1186 @wackerow any thoughts?

**Comment by minimalsm** [comment]
Todo: confirm these are mainnet-merge ready before linking out

File: src/content/developers/docs/nodes-and-clients/run-a-node/index.md (hunk 1 above)
Comment (jmcook1186) [reply src/content/developers/docs/nodes-and-clients/run-a-node/index.md]:
```suggestion
- [ETHStaker guides on running validators on testnets](https://github.com/remyroy/ethstaker#guides) – _ETHStaker, updated regularly - non-validating node operators can simply omit sections on staking and starting validators_
- [Ethereum Staking Guides](https://github.com/SomerEsat/ethereum-staking-guides) _– Somer Esat, updated regularly - non-validating node operators can simply omit sections on staking and starting validators_
```

**Comment by sandakersmann** [comment]
> Todo: confirm these are mainnet-merge ready before linking out

I have confirmed this.

**Review by jmcook1186 — APPROVED** [review]
fine by me - these are up to date for mainnet merge - I think they're a useful resource. Non-validating nodes can just skip the sections on staking and adding validators.

**Comment by wackerow** [comment]
Just noting we link out to this already in the StakingGuides component (specifically to https://github.com/SomerEsat/ethereum-staking-guide which redirects to the above spelling... we could update this separately)
Excellent set of guides though, I'm okay with this also living in this dev docs page

**Pull request merged**